app/initializers/register-select-helper.js

```diff
@@ -0,0 +1,13 @@
+import ENV from '../config/environment';
+import registerSelectHelper from 'emberx-select/helpers/register-select-helper';
```

Comment:
This should probably be in `test-support` or some other package that isn't included into the app in production. I don't really know how to do that though. Maybe @fivetanley does?